wd_util.c

```diff
@@ -246,24 +241,23 @@ int wd_get_msg_from_pool(struct wd_async_msg_pool *pool, int index, void **msg)
 			return -WD_EBUSY;
 	}
 
+	p->tail = (idx + 1) % msg_num;
```

Comment:
1. **head** & **tail** were both defined without usage. Now only **tail** is enabled in this patch. But **head** is still no used. **head** and **tail** are always in pair. I suggest to rename it to **last**. If **tail** is used, user always think where is **head** and how to use it.
2. It's not good to update the **tail** directly. The assignment is composed by a few instructions. It could be switched to other threads at any time. I think that we could use atomic store on updating **tail**. Same on loading **tail** above.

Reply:
**tail** is also fine. Please help to remove **head** since it's not used at all.